locale/en/LC_MESSAGES/django.po

```diff
@@ -2517,82 +2517,82 @@ msgid ""
 msgstr ""
 
 #. Translators: Do not translate.
-#: TWLight/users/models.py:793
+#: TWLight/users/models.py:794
```

Comment:
What does the number incrementing mean? Is this just a reference to a line of code?

Reply:
Yep, there is an entry for every place the string appears. These comments are generated automatically.